packages/cyberstorm-forms/src/components/FormTextInput.tsx

```diff
@@ -13,17 +14,24 @@ export type FormTextInputProps<
   schema: Schema;
   name: Path<z.infer<Schema>>;
   placeholder?: string;
+  existingValue?: string;
 };
 export function FormTextInput<
   Schema extends ZodObject<Z>,
   Z extends ZodRawShape
->({ name, placeholder }: FormTextInputProps<Schema, Z>) {
+>({ name, placeholder, existingValue }: FormTextInputProps<Schema, Z>) {
   const {
     field,
     fieldState: { isDirty, invalid, error },
     formState: { isSubmitting, disabled },
   } = useController({ name });
 
+  if (existingValue) {
```

Comment:
As stated in a comment for `FormSelect`, I'd prefer this would be handled via RHF rather than manually in each component. If that's not viable, at least rename the prop to defaultValue to be consistent with prop name of `FormSelect`